src/components/dashboard.js

```diff
@@ -1,7 +1,214 @@
+              </div>
+
+              <div class="get__started col-lg-4 col-md-4 col-sm-8 d-flex align-items-end">
+                <button class="btn btn-block btn-primary mr-5">
+                  Get Started
+                </button>
+              </div>
+            </div>
+            {/* <!-- <img src="" class="background-vector img-fluid" alt="" srcset=""> --> */}
+            <span id="close__user__info">x</span>
+          </div>
+          <div class="cards">
+            <div class="card">
+              <div class="card-body mentor-card">
+                <div class=" card-image card-image-1">
+                  <img class="img-fluid" src="img/Group.png" />
+                </div>
+                <h6 class="card-subtitle">
+                  <span>1</span>
+                  <p>Approved Mentor</p>
+                </h6>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body resource-card">
+                <div class="card-image card-image-2">
+                  <img class="img-fluid" src="img/newspaper.png" />
+                </div>
+                <h6 class="card-subtitle">
+                  <span>2</span>
+                  <p>New Resources</p>
+                </h6>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body task-card">
+                <div class="card-image card-image-3">
+                  <img class="img-fluid" src="img/newspaper2.png" />
+                </div>
+                <h6 class="card-subtitle">
+                  <span>1</span>
+                  <p>Pending Task</p>
+                </h6>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body progress-card">
+                <img class="card-title" src="img/Progress Bar.png" />
+                <p class="card-subtitle">HTML Fundamentals II</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="pending-tasks-wrap mt-5">
+            <div class="header">
+              <p class="p-2 pl-5">Pending Tasks</p>
+            </div>
+
+            <div class="solid-bar"></div>
+
+            <div class="pending-tasks pb-5">
+              <div class="pending-task"></div>
+
+              <div class="pending-task"></div>
+
+              <div class="pending-task"></div>
+
+              <div class="pending-task"></div>
+            </div>
+
+            <div class="pagination">
+              <nav aria-label="...">
+                <ul class="pagination pagination-sm">
+                  <li class="page-item" aria-current="page">
+                    <span class="page-link active-page">
+                      1<span class="sr-only">(current)</span>
+                    </span>
+                  </li>
+                  <li class="page-item">
+                    <a class="page-link" href="#">
```

Comment:
Codacy found an issue: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5730659)